fastify.js

```diff
@@ -351,6 +352,8 @@ function fastify (options) {
   avvio.on('start', () => (fastify[kState].started = true))
   fastify[kAvvioBoot] = fastify.ready // the avvio ready function
   fastify.ready = ready // overwrite the avvio ready function
+  fastify.printPlugins = avvio.prettyPrint.bind(avvio)
```

Comment:
I think exposing [`avvio.toJSON()`](https://github.com/fastify/avvio/#avviotojson) would be more flexible
Then we could just create a plugin that adds the `printPlugins` decorator function

Do you agree?

Reply:
I think it depends on if you consider this functionality core or not. I think because it is useful only really for debugging, and thusly only really rarely when you have a suspicion the require order is wrong, it'd be pretty annoying to have to go find and install the plugin that does a one line thing instead of it just being built in. I am not exactly sure what the rubric is for inclusion in core, but this seems like a convenience feature with no real cost so I will vote for it's inclusion.